src/components/Design/SVG/ChevronDown.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export default function ChevronDown() {
```

Comment:
You don't need to create a separate component if you only want to import an SVG-file with no extra functions or styling.

You can just import it in the component where you want to use it like this: `import {ReactComponent as ChevronDown} from './chevron.svg';
`

Reply:
I will do that from now on, thank you for the hint! The component was a leftover from an experiment on the weekend and is now deleted.